CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+# Contribution guidelines
+
+Thank you for investing your time and considering contributing to Retis! :tada:
+
+This guide documents the process to get changes merged into this project, what
+one should expect, as well as prerequisites for pull-requests and patches to be
+considered acceptable.
+
+### Table of contents
+
+* [Reporting bugs](#reporting-bugs)
+* [Suggesting new features](#suggesting-new-features)
+* [Code contribution](#code-contribution)
+
+## Reporting bugs
+
+Following the guidelines will help the community understand bug reports, which
+in turns will help in reproducing, triaging and fixing those bugs.
+
+### Before submitting the report
+
+1. Check for similar opened or closed issues. If a similar issue is still
+   opened, please add a comment there.
+1. Check the version used when the bug was triggered. If it was not the latest,
+   please reproduce with it.
+
+### Submitting a good bug report
+
+1. Use a clear, concise and descriptive title.
+1. Describe how to reproduce the bug, in clear exact steps, with examples. The
+   community should not have to guess extra steps, configuration, prerequisites
+   and spend lot of time gathering this information to work on fixing a bug.
+1. How reliable are those steps to reproduce the issue?
+1. Describe the outcome of those steps.
+1. Then describe what would have been expected and why.
+1. Don't make vague reports. If it is performances related, include numbers.
```

Comment:
I would split the last item. The "performance numbers" seems important enough to stay on its own.

Reply:
ok, I misinterpreted the intent then :)
What about just going with:
"If it is performances related, include numbers."